eng/Generate.ps1

```diff
@@ -108,7 +108,8 @@ $projectNames =
     'Azure.AI.FormRecognizer',
     'Azure.Storage.Tables',
     'Azure.Management.Storage',
-    'Azure.Network.Management.Interface'
+    'Azure.Network.Management.Interface',
+    'Azure.Storage.Blobs'
```

Comment:
This is where you need to add the new sample's name when you create a new sample.